### PR TITLE
Fix duplicates in detailed node activity

### LINKED ISSUE #39890: Release test rllib_learning_tests_a3c_tf.aws failed
Release test **rllib_learning_tests_a3c_tf.aws** failed. See https://buildkite.com/ray-project/release-tests-branch/builds/2194#018ad2dc-0968-4747-a5eb-5af95f1290d3 for more details.

Managed by OSS Test Policy

### PR DESCRIPTION
**Pull request by vitsai**
Overwrite the field instead of appending to it.

## Why are these changes needed?

https://github.com/ray-project/ray/issues/39850

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by rickyyx — COMMENTED** [review]
Let's add e2e test?

Also, should we just also add the idle time into this PR?

File: python/ray/tests/test_autoscaler_e2e.py [hunk 1]
```diff
@@ -14,6 +14,50 @@
...
+                return None
+
+        actor = Actor.remote()
+        ray.get(actor.ping.remote())
+
+        occurrences = 1 if enable_v2 else 0
+        assert (
+            subprocess.check_output("ray status --verbose", shell=True)
+            .decode()
+            .count("Resource: CPU currently in use.")
```
Comment (rickyyx) [inline python/ray/tests/test_autoscaler_e2e.py:54]: Can we test that the `bundle_group_xxx` for placement groups resources are not showing up here?

**Review by rickyyx — APPROVED** [review]
Thanks for the quick fix and iteration!

**Comment by rkooo567** [comment]
can you make sure to run release tests that are failing to verify it is fixed before merging it?

**Comment by rkooo567** [comment]
I ran one test here; https://buildkite.com/ray-project/release-tests-pr/builds/54054

**Comment by rickyyx** [comment]
It passses: https://buildkite.com/ray-project/release-tests-pr/builds/54054

**Pull request merged**